SConstruct

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConstruct("deploy_testing_distrib")
```

Comment:
Yes, the `.build` files are there to teach `sconsUtils` how to link against this package. That's obviously not necessary for metapackages.